bspump/elasticsearch/connection.py

```diff
@@ -109,7 +110,11 @@ async def upload(self, url, session, ssl_context, timeout):
 		except OSError as e:
 			# This means that there was a hard error such as network or DNS failure
 			# Likely no communication took place with ElasticSearch
-			L.warn("{}".format(e))
+			L.warn(
```

Comment:
In this repository we use `L.warning` a lot... And `L.warn` on several places, basically only in a couple of files. Is it on purpose?

Reply:
It should be L.warning. ... L.warn is obsoleted.

Reply:
@Plesoun maybe it's better to correct in this PR then?